lib/doorkeeper/config.rb

```diff
@@ -46,7 +46,7 @@ def initialize(&block)
       end
 
       def build
-        @config
+        @config.valid_subset
```

Comment:
I personally prefer something like just `validate`, or `validate!`

Reply:
That's fine with me, changed to `validate`